ush/gaussian_sfcanl.sh

```diff
@@ -97,9 +97,10 @@
 ################################################################################
 
 CASE=${CASE:-C768}
-res=$(echo $CASE | cut -c2-)
-LONB_CASE=$((res*4))
-LATB_CASE=$((res*2))
+CASE_HIST=${CASE_HIST:-C768}
+resh=$(echo $CASE_HIST | cut -c2-)
```

Comment:
```suggestion
CASE_HIST=${CASE_HIST:-${CASE}}
resh=${CASE_HIST:1}
```

For clarity, we should note that the interpolation is from surface restarts at `$CASE` resolution to the Gaussian equivalent of `$CASE_HIST`.

The doc-block of this script is outdated.  Just noting, nothing to do for this PR.